content/en/docs/faq/migrating-to-hypershift.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Migrating to HyperShfit FAQ"
+description: FAQ on migrating CI tests from use regular AWS clusters to HyperShift clusters.
+---
+This documents is an FAQ for migrating CI tests from use regular AWS clusters to HyperShift clusters.
+
+## Where is the workflow?
+Check the [hypershift-hostedcluster-workflow](https://steps.ci.openshift.org/workflow/hypershift-hostedcluster-workflow) in step-registry: 
+
+## What does the workflow do?
+The workflow we are providing will claim a hosted cluster from DPTP-managed HyperShift deployment.
+
+## Who can use it?
+Anyone who just want a cluster in AWS
+
+## Who may not use it?
+Anyone who is developing and testing OCP components, except `console` and `monitoring`
+
+## What are the benefits?
+1. Faster. The cluster can usually be ready in ~15min.
+2. Cheaper. There are only worker nodes in your AWS account. Control-plane is hosted in our management cluster.
+
+## Which version of OpenShift is supported?
+4.12.z to 4.13 master
```

Comment:
What is the plan for this as time goes by? Do we have to update the versions here regularly?

Reply:
We probably will, once HyperShift team figured out their release schedule.